Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-ruby '2.2.1'
+ruby '2.1.2'
```

Comment:
For anyone curious: the step down in ruby versions was for compatibility with Traveling Ruby so this could run on [AWS Lambda](https://github.com/eeeschwartz/ruby-on-lambda). Turns out it's a no-go since native gems like nokogiri are not supported on Lambda yet, so I'll bump the ruby version up in the next PR.